### PR TITLE
lgtm fixes

### PR DESCRIPTION
**Pull request by ThomasWaldmann**
https://lgtm.com/projects/g/borgbackup/borg/alerts/

just fixed the easy stuff. any ideas about the rest?

### CODE REVIEW

File: src/borg/constants.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 # MAX_OBJECT_SIZE = <20 MiB (MAX_DATA_SIZE) + 41 bytes for a Repository PUT header, which consists of
 # a 1 byte tag ID, 4 byte CRC, 4 byte size and 32 bytes for the ID.
 MAX_OBJECT_SIZE = MAX_DATA_SIZE + 41  # see LoggedIO.put_header_fmt.size assertion in repository module
-assert MAX_OBJECT_SIZE == 20971520 == 20 * 1024 * 1024
+assert MAX_OBJECT_SIZE == 20 * 1024 * 1024  # == 20971520
```
Comment (enkore) [inline src/borg/constants.py]: -1 to this.

File: src/borg/repository.py [hunk 2]
```diff
@@ -642,7 +642,8 @@ def complete_xfer(intermediate=True):
             # get rid of the old, sparse, unused segments. free space.
             for segment in unused:
                 logger.debug('complete_xfer: deleting unused segment %d', segment)
-                assert self.segments.pop(segment) == 0, 'Corrupted segment reference count - corrupted index or hints'
+                count = self.segments.pop(segment)
+                assert count == 0, 'Corrupted segment reference count - corrupted index or hints'
                 self.io.delete_segment(segment)
```
Comment (enkore) [inline src/borg/repository.py]: good

File: src/borg/constants.py (hunk 1 above)
Comment (ThomasWaldmann) [inline src/borg/constants.py]: well, if we keep as is, lgtm.com will have that warning in every report...

Comment (enkore) [inline src/borg/constants.py]: Then rather remove the explicit number entirely. (Though I doubt the usefulness of any static analysis unable to suppress false positives.)

**Pull request merged**